Taskfile.yaml

```diff
@@ -478,22 +478,11 @@ tasks:
     vars:
       clogs:
         sh: |
-          git diff $(git merge-base origin/main HEAD): --numstat | awk '{if ($3 ~ /^.changelog/ && $3 ~ /.yaml$/ && $3 !~ /TEMPLATE[.]yaml$/) { print $3 }}' | sed 's-.changelog/--g' | xargs
-    preconditions:
-      - sh: bash -c '[[ "" != "{{.clogs}}" ]]'
-        msg: "No changelog files found in PR"
+          git diff $(git merge-base origin/main HEAD): --no-renames --name-status -- .changelog/ | awk '{if ($1 != "D" && $2 ~ /.yaml$/ && $2 !~ /TEMPLATE[.]yaml$/) { print $2 }}' | sed 's-.changelog/--g' | xargs
+    # Do not run if there are no changelog files
+    status:
+      - "[ '{{.clogs}}' == '' ]"
     cmds:
       - cmd: pip show pyyaml 2>&1 >/dev/null || pip install pyyaml
         silent: true
       - python .changelog/changelog-gen.py {{.clogs}}
```

Comment:
`task changelog-pr` behavior change 

Previous
   1. No changelog added => preconditions fail it
   2. Changelog was deleted => clogs variable passed to changelog-gen.py that fails it when python script fails to find file

Changes:
  1. No changelog added => status returns early
  2. Changelog was deleted -- Cannot happen because `awk` is filtering by delete files `($1 != "D")`

Reply:
`task changelog-pr` Only failure can occur when added changelog is in invalid format

github workflow `changelog` fails on 2 condition
1. No new changelog added
2. changelog added is in incorrect format
3. **Fixed in this PR** ~~changelog deleted between `origin/main` and HEAD~~